build/AzurePipelinesTemplates/MUX-InstallDotNetSDK-Steps.yml

```diff
@@ -0,0 +1,27 @@
+parameters:
+  sdkVersion: 5.0.404
+
+steps:
+  - powershell: Invoke-WebRequest "https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.ps1" -OutFile "$(Agent.TempDirectory)\dotnet-install.ps1"
```

Comment:
Can we use the UseDotNet@2 task instead here?

See [https://github.com/microsoft/microsoft-ui-xaml/blob/f9827bb2086f0b86d5a8a3caa264527cf6acf472/build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml#L18](https://github.com/microsoft/microsoft-ui-xaml/blob/f9827bb2086f0b86d5a8a3caa264527cf6acf472/build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml#L18)